test/monotonic_resource.cpp

```diff
@@ -14,12 +15,108 @@
 
 #include "test_suite.hpp"
 
+#include <memory>
+#include <iostream>
+
 namespace boost {
 namespace json {
 
 class pool_test
```

Comment:
This should say `monotonic_resource_test`

Reply:
Copy